rllib/env/pettingzoo_env.py

```diff
@@ -0,0 +1,191 @@
+from ray.rllib.env import MultiAgentEnv
+
+class PettingZooEnv(MultiAgentEnv):
+    """An interface to the PettingZoo MARL environment library.
```

Comment:
This isn't passing the linter: https://travis-ci.com/github/ray-project/ray/jobs/356890498

You can run `scripts/format.sh` to auto format most of the things (still have to fix line too long manually).